android/app/src/main/java/com/tradingview/android/MainActivity.kt

```diff
@@ -0,0 +1,22 @@
+package com.tradingview.android
+
+import android.os.Bundle
+import android.support.v7.app.AppCompatActivity
+import android.webkit.WebViewClient
+import kotlinx.android.synthetic.main.activity_main.*
+
+class MainActivity : AppCompatActivity() {
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_main)
```

Comment:
I would like to remove xml file and create WebView by Kotlin code.
Something like:
```
override fun onCreate(savedInstanceState: Bundle?) {
    super.onCreate(savedInstanceState)
    initWebView()
}

fun initWebView() {
    val webView = WebView(this)
    webView.settings.javaScriptEnabled = true
    webView.settings.allowFileAccessFromFileURLs = true
    webView.loadUrl("file:///android_asset/mobile_white.html")
    setContentView(webView)
}
```